packages/asset-uploader/src/index.ts

```diff
@@ -0,0 +1,63 @@
+import { DEFAULT_UPLPOAD_PATH } from "./constants";
+import path from "path";
+import {
+  unstable_createFileUploadHandler,
+  unstable_parseMultipartFormData,
+} from "@remix-run/node";
+import { s3UploadHandler } from "./targets/s3/handler";
+import { uploadToS3 } from "./targets/s3/uploader";
+import { uploadToDisk } from "./targets/disk/upload";
+
+export const uploadAsset = async ({
+  request,
+  projectId,
+  db,
+  dirname,
+}: {
+  request: Request;
+  projectId: string;
+  db: unknown;
+  dirname: string;
+}) => {
+  const IS_S3_UPLOAD =
```

Comment:
if that's a boolean it should be camel case and Boolean(....)

Reply:
also maybe if some of those env variables are not present, we could show an error at the validator in the env, because if all 3 only work together as one thing, it should be obvious that all 3 need to be defined in the env

Reply:
Done and done!

It was actually four so I am glad I checked